pkg/common/env/v1alpha1/env.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2018 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"os"
+	"strconv"
+	"strings"
+)
+
+// ENVKey is a typed string to represent various environment keys used by this
+// binary
+type ENVKey string
+
+const (
+	// OpenEBSNamespace is the environment variable to get openebs namespace
+	//
+	// This environment variable is set via kubernetes downward API
+	OpenEBSNamespace ENVKey = "OPENEBS_NAMESPACE"
```

Comment:
Let these constants be defined close to the caller

Reply:
done.

Reply:
sure.